backend-api/run-tests.sh

```diff
@@ -7,6 +7,7 @@ remove_quotes() {
   echo "$1" | tr -d '"'
 }
 export PROXY_API_URL=$(remove_quotes "$CFN_ProxyApiUrl")
+export PRIVATE_API_URL=$(remove_quotes "$CFN_PrivateApiUrl")
```

Comment:
I really want to get away from calling this Private API url in the not to distant future. Not for this PR. Private describes its implementation rather than it's purpose.